From 438533ce9da1df0b7c7914e64b39ffdc1da1ab79 Mon Sep 17 00:00:00 2001
From: Brett Warren <brett.warran@arm.com>
Date: Thu, 8 Oct 2020 10:03:25 +0100
Subject: [PATCH] make: remove -Wmissing-noreturn for clang

When compiling when clang, -Wmissing-noreturn causes an error because
of non-compliant code. This option is removed to workaround this.

Upstream-Status: Pending
Changed-Id: 71cb511904547d790d1ea98f93bf8e5a6afcb36d
Signed-off-by: Brett Warren <brett.warren@arm.com>
---
 host/xtest/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/xtest/Makefile b/host/xtest/Makefile
index 3c206b0..96746de 100644
--- a/host/xtest/Makefile
+++ b/host/xtest/Makefile
@@ -169,7 +169,7 @@ CFLAGS += -Wall -Wcast-align -Werror \
 	  -Werror-implicit-function-declaration -Wextra -Wfloat-equal \
 	  -Wformat-nonliteral -Wformat-security -Wformat=2 -Winit-self \
 	  -Wmissing-declarations -Wmissing-format-attribute \
-	  -Wmissing-include-dirs -Wmissing-noreturn \
+	  -Wmissing-include-dirs \
 	  -Wmissing-prototypes -Wnested-externs -Wpointer-arith \
 	  -Wshadow -Wstrict-prototypes -Wswitch-default \
 	  -Wwrite-strings \
-- 
2.17.1

